src-docs/src/views/page/page_example.js

```diff
@@ -132,7 +154,75 @@ export const PageExample = {
       ),
     },
     {
-      title: 'Page header',
+      title: 'A simple page layout with content only',
+      source: [
+        {
+          type: GuideSectionTypes.JS,
+          code: pageContentOnlySource,
+        },
+        {
+          type: GuideSectionTypes.HTML,
+          code: pageContentOnlyHtml,
+        },
+      ],
+      text: <p>We can further simplify pages by only showing the content.</p>,
+      demo: (
+        <div className="guideDemo__highlightLayout">
+          <PageContentOnly />
+        </div>
+      ),
+    },
+    {
+      title: 'A simple page layout with content centered',
+      source: [
+        {
+          type: GuideSectionTypes.JS,
+          code: pageContentCenterSource,
+        },
+        {
+          type: GuideSectionTypes.HTML,
+          code: pageContentCenterHtml,
+        },
+      ],
+      text: (
+        <p>
+          The page content can be optionally centered either vertically or
+          horizontally. This is useful for various empty states.
+        </p>
+      ),
+      demo: (
+        <div className="guideDemo__highlightLayout">
+          <PageContentCenter />
+        </div>
+      ),
+    },
+    {
+      title: 'A simple page layout with content centered in a full layout',
+      source: [
+        {
+          type: GuideSectionTypes.JS,
+          code: PageContentCenterWithSideBarSource,
+        },
+        {
+          type: GuideSectionTypes.HTML,
+          code: PageContentCenterWithSideBarHtml,
+        },
+      ],
+      text: (
+        <p>
+          Centering the content can happen regardless of layout configuration.
+          In this example, we&rsquo;re centering within a complex sidebar
+          layout.
+        </p>
+      ),
+      demo: (
+        <div className="guideDemo__highlightLayout">
+          <PageContentCenterWithSideBar />
+        </div>
+      ),
+    },
+    {
+      title: 'The EuiPageHeader component in more detail',
```

Comment:
We don't use the `Eui` style of component naming in these section titles. Just trying to stay consistent and cognizant of how it will appear in the side nav and with search.

```suggestion
      title: 'The page header in detail',
```